RELEASE_NOTES.md

```diff
@@ -1,5 +1,8 @@
 # Release notes
 
+## New in git-machete 3.2.2
```

Comment:
Whoops that's more of 3.3.0 since command-line interface has changed... see https://github.com/VirtusLab/git-machete/blob/master/CONTRIBUTING.md#versioning